examples/cognito/identityprovider.yaml

```diff
@@ -0,0 +1,52 @@
+---
+apiVersion: cognitoidentityprovider.aws.crossplane.io/v1alpha1
+kind: IdentityProvider
+metadata:
+  name: example-provider
+spec:
+  forProvider:
+    region: eu-central-1
+    providerType: OIDC
+    # You'll need to update this to details specified by your external identity provider
+    providerDetails:
+      client_id: foo000000000000000000
+      client_secret: bar000000000000000000
```

Comment:
think providerDetails should be a kubernetes secret ?! like your other example - i would skip normal providerDetails because of secret content ?